feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go

```diff
@@ -136,6 +136,8 @@ func setPolicyPeerGroup(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, policy
 // isConverged function is used to check if ATE has received all the prefixes.
 func isConverged(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, ap *ondatra.Port) {
 
+	// Add 10 second timer for BGP update to propagate
```

Comment:
Can you help elaborate why current logic (without this delay) is not good enough? thanks.

Reply:
Current version of isConverged() assumes that an implementation does withdrawal and re-advertisement of the routes after the new policy (with modified MED) is applied - it monitors the number of received prefixes on the OTG ports

In some implementations, including ours, there's no withdrawal step - we just send updates with the new attribute values. Hence this function will always think that the convergence is done, even when it's not